alternatives.tex

```diff
@@ -1,7 +1,13 @@
 
 \subsubsection{Alternatives to Distributed machine learning}
 The most popular solution to increasing computation power for Machine
-learning is currently distributing the workload over a large amount of machines. However, there are other, more traditional ways to increase the available computation power. These ways include the use of Application Specific Integrated Circuits (ASICs) or special multi-core computer architectures.
+learning is currently distributing the workload over a large amount of machines. However, there are other, more traditional ways to increase the available computation power. Such as the use of dedicated Graphics processing units (GPUs), the use of Application Specific Integrated Circuits (ASICs) or special multi-core computer architectures.
+
+\paragraph{GPUs in Machine Learning}
+The use of GPUs in Machine Learning is already a very common method. This is due to the nature of the data which makes up Machine Learning problems. A GPU is extremly fast at making multiple small calculations on a batch of data. Luckily the data that is requiered by Machine Learning algorithms such as Neural Networks and Unsupervised Learning is of this type. This means that GPUs have had a lot of success with speeding up these calcultions as was found by Meuth \ref{Meut2007} who reported a up to 200x speed up over conventional CPUs.
```

Comment:
"requiered" -> "required"

Reply:
"calcultions" -> "calculations, "

Reply:
"reported a up to 200x speed up over" -> "reported a speedup factor of over 200 compared to"

Reply:
Speedup ref is outdated, growth has stabilized probably